packages/ember-simple-auth/tests/unit/internal-session-test.js

```diff
@@ -808,11 +804,4 @@ describe('InternalSession', () => {
       });
     });
   });
-
-  it('does not share the content object between multiple instances', function() {
```

Comment:
It didn't make sense to me since there's only 1 `session:main` in the app anyway?